app/components/courses/international_students_component.rb

```diff
@@ -12,13 +10,43 @@ def provider
       @provider ||= @course.provider
     end
 
+    def right_required
+      if @course.salaried?
+        'right to work'
+      else
+        'right to study'
+      end
+    end
+
     def visa_sponsorship_status
       if !@course.salaried? && provider.can_sponsor_student_visa
-        "We can sponsor #{govuk_link_to('Student visas', TRAIN_TO_TEACH_URL)}, but this is not guaranteed.".html_safe
+        "<p class=\"govuk-body\">If you do not already have the right to study in the UK for the duration of this course, you may need to apply for a Student visa.</p>
```

Comment:
I wonder whether you should use [tag helpers]( https://apidock.com/rails/ActionView/Helpers/TagHelper/tag) here since it is an `.rb` file and even the en.yml locales for the text. @avinhurry @frankieroberto do you think this is a good idea?

Reply:
discussed offline